docs/reference/src/code/examples/access-control/ownership/src/main.sw

```diff
@@ -1,8 +1,5 @@
 contract;
 
-// ANCHOR: identity
```

Comment:
Should we keep the anchor? Looks like we are still using it

Reply:
Added back in! @kayagokalp